rebasehelper/tests/public_api/test_specfile.py

```diff
@@ -48,6 +49,18 @@ def test_update_changelog(self, spec_object):
         assert spec_object.update_changelog('test2') is None
         assert spec_object.update_changelog(changelog_entry='test') is None
 
+    def test_get_new_log_with_non_c_locale(self, spec_object):
```

Comment:
Could you please move the test to `rebasehelper/tests/test_specfile.py`? Public API tests are meant to test the stability of the interface of certain methods, not the functionality.

Reply:
Makes sense. Done in the FIXUP.